modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/dynamic_include/bottom.jsp

```diff
@@ -59,28 +59,38 @@ sharingURL.setWindowState(LiferayWindowState.POP_UP);
 		Sharing,
 		'share',
 		function(classNameId, classPK, title, namespace, refreshOnClose) {
-			var sharingURL = new Liferay.PortletURL.createURL('<%= sharingURL.toString() %>');
+			var parameters = {
+				'classNameId': classNameId,
+				'classPK': classPK
+			}
```

Comment:
Missing semicolon here?

We might also want to get rid of the quotes for the object keys as they don't seem necessary...